sourcegraph/cody
# Smart Apply: Open the correct document when in a different view column

Closes CODY-3242

## Description

Fixes a bug where, when Cody Chat was in a different view column, we would open the document again even though it was already visible.



## Test plan

1. Open Cody Chat in a split editor view (side by side with the open file)
2. Ask Cody to make a change to the open file
3. Click "Apply" from the chat view
4. Expect that the edit continues in the already visible file, it does not open a duplicate file
